README.md

```diff
@@ -0,0 +1,114 @@
+# Scalar Admin for Kubernetes
+
+Scalar Admin for Kubernetes is a tool that creates a paused state for Scalar products (for example, ScalarDB and ScalarDL) in a Kubernetes environment. You can use such a paused state to take backups easily and consistently across multiple diverse databases.
```

Comment:
When we mention `Scalar products`, do we mean just ScalarDB and ScalarDL? If so, I'd like to change that part as follows:

```suggestion
Scalar Admin for Kubernetes is a tool that creates a paused state for ScalarDB or ScalarDL in a Kubernetes environment. You can use such a paused state to take backups easily and consistently across multiple diverse databases.
```

Reply:
> do we mean just ScalarDB and ScalarDL?

Yes. At this time, we mean ScalarDB and ScalarDL as a target of Scalar Admin for Kubernetes.
I will apply the suggestion. Thank you!